types/w3c-css-typed-object-model-level-1/index.d.ts

```diff
@@ -0,0 +1,290 @@
+// Type definitions for non-npm package css-typed-object-model-level-1 20180410.0
+// Project: https://www.w3.org/TR/css-typed-om-1/
+// Definitions by: Dmitry Guketlev <https://github.com/yavanosta>
```

Comment:
Q: should this be unowned package then?
```suggestion
```

Reply:
This failed CI -- I'll just put my name on it.